lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -636,6 +636,16 @@ def write(chunk)
                           record.delete(@trace_key)
                         end
 
+          ts_secs = begin
+                      Integer ts_secs
+                    rescue ArgumentError, TypeError
+                      ts_secs
+                    end
+          ts_nanos = begin
+                       Integer ts_nanos
+                     rescue ArgumentError, TypeError
+                       ts_nanos
```

Comment:
I've tried that earlier, but it caused the tests to fail. Our tests expect the code to preserve the value if it's not an integer.